src/lib/components/vegachart/VegaChart.component.js

```diff
@@ -26,6 +26,13 @@ function VegaChart({ id, spec }: Props) {
   const brandText =
     themeContext && themeContext.brand && themeContext.brand.text;
 
+  let tooltipTheme;
+  if (currentBackgroundColor === "#ffffff") {
+    tooltipTheme = "light";
+  } else {
+    tooltipTheme = "dark";
+  }
+
```

Comment:
It is not a robust way. you may need to add `tooltipTheme ` to the props. There is another way to customize the tooltip instead of using vega default theme ?

Reply:
I added a new prop in `vegachart` component.

Actually, we can customize the theme for tooltip by override the CSS.  The only way that works for me is create a CSS file and import it in vegachart. I am wondering do you have better ideas?

Please find the following link that explains how to customize the tooltip by using vega-tooltip.
[doc - customize tooltip](https://github.com/vega/vega-tooltip/blob/master/docs/customizing_your_tooltip.md)

Reply:
ok! according to the doc, `theme` could be anything (light, dark, custom...) So it is up to the user to come with there own CSS and configure `theme` to override the theme.